### PR TITLE
Add tutorials to ReadTheDocs

### PR DESCRIPTION
**Pull request by 10220**
This adds three tutorials to ReadTheDocs:

- Using the `AsymptoticBondiData` class
- Using the `WaveformModes`/`WaveformGrid` class
- Performing extrapolation

These tutorials are not exhaustive but provide enough information to get a new user started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ If changes need to be made to the requirements or anything else in `recipe/meta.
 
 ## Documentation
 
-Automatically-generated API documentation is available on [Read the Docs: scri](https://scri.readthedocs.io/).
+Tutorials and automatically-generated API documentation are available on [Read the Docs: scri](https://scri.readthedocs.io/).
```
[the remark this answers is not in the document]
Comment (10220) [reply README.md]: :laughing:

**Comment by 10220** [comment]
I've built the docs on my machine, would it be easier if emailed you the `_build` directory so you can open it up on your machine?

**Comment by moble** [comment]
Oh, no, that's okay.  That hyphen change triggered [a build.](https://scri--56.org.readthedocs.build/en/56/)

**Comment by 10220** [comment]
I forgot, this PR addresses issue #47.

**Pull request merged**